subprojects/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/JacocoPluginMultiVersionIntegrationTest.groovy

```diff
@@ -34,6 +34,7 @@ class JacocoPluginMultiVersionIntegrationTest extends JacocoMultiVersionIntegrat
 
     def setup() {
         javaProjectUnderTest.writeSourceFiles()
+        executer.noDeprecationChecks()
```

Comment:
Yep, when running on Java 7 the assertion `file(REPORTING_BASE).listFiles().collect { it.name } as Set == ["jacoco", "tests"] as Set` would fail. Do you have a better idea?